metadata.json

```diff
@@ -1,15 +1,12 @@
 {
-  "name": "puppetlabs-corosync",
+  "name": "puppet-corosync",
   "version": "0.7.0",
```

Comment:
why do we have to commit to the API?
semver just means that if we break it (for good reasons, like, because it's 💩), we just bump the MAJOR version.